src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.css

```
@@ -21,3 +23,14 @@
 .input-icon {
   composes: input-icon from '../custom-field-input-text/custom-field-input-text.css';
 }
+
+.popup-container {
+  composes: container from '../listbox/listbox.css';
+  background-color: var(--white);
+  box-sizing: border-box;
+  max-height: 160px;
+  overflow: scroll;
```

I think we want
```suggestion
  overflow-y: scroll;
```

Looking at the DRD, items in the dropdown that are too long to render on one line should use an ellipsis.

---

I'll verify with design.

---

In either case, we don't want the x scrollbar. 😄

---

Looks like the default is to truncate already without modifying the CSS property. I think the behavior is built into the Listbox.

---

@juanca Still want to make the `overflow` change? We don't need the horizontal scroll.